lib/flipper/ui/expression_param_parser.rb

```diff
@@ -0,0 +1,40 @@
+module Flipper
+  module UI
+    # Internal: Used to parse expressions from the UI into a format that can be
+    # used by Flipper::Expression.build
+    class ExpressionParamParser
```

Comment:
I tried to get the request on the form submission as close to the expression AST as I could to reduce the amount of parsing. This was the best I could do